docs/DeveloperGuide.md

```diff
@@ -234,29 +234,44 @@ _{Explain here how the data archiving feature will be implemented}_
 
 ### Product scope
 
-**Target user profile**:
+**Target user profile**: Business Owners who needs to keep track of their stock inventory
 
-* has a need to manage a significant number of contacts
+* needs to manage purchasing appropriate amounts of raw materials based on usage
+* wants to be updated on raw materials stock level on a daily basis
```

Comment:
Should we add the app's ability to handle table reservations and pending deliveries? (like in Mock UI) 
Since this is the final product's aim

Reply:
just pushed! take a look